### PR TITLE
Add Formulatepro v0.0.6

### PR DESCRIPTION
**Pull request by shimeike**
Updated to link to github instead of googlecode per discussion.
Thanks!

### CODE REVIEW

**Comment by fanquake** [comment]
Can you please squash your commits

**Comment by shimeike** [comment]
Ok, done (I think)

File: Casks/formulatepro.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'formulatepro' do
+  version '0.0.6'
+  sha256 '9a5c37bad02a9dea7448e4ebe6fc6b0887efa05b0530603b9be1e5b0b3db2542'
+
+  url 'https://github.com/shimeike/formulatepro/releases/download/v0.0.6a/FormulatePro-0.0.6.dmg'
+  name 'formulatepro'
+  homepage 'https://github.com/adlr/formulatepro'
```
Comment (vitorgalvao) [inline Casks/formulatepro.rb]: Has to be changed to `https://github.com/shimeike/formulatepro`.

File: Casks/formulatepro.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'formulatepro' do
```
Comment (vitorgalvao) [inline Casks/formulatepro.rb]: Has to be changed to `shimeike-formulatepro`, to make it clear it is a fork.

File: Casks/formulatepro.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  version '0.0.6'
+  sha256 '9a5c37bad02a9dea7448e4ebe6fc6b0887efa05b0530603b9be1e5b0b3db2542'
+
+  url 'https://github.com/shimeike/formulatepro/releases/download/v0.0.6a/FormulatePro-0.0.6.dmg'
```
Comment (vitorgalvao) [inline Casks/formulatepro.rb]: Please add a `appcast 'https://github.com/shimeike/formulatepro/releases.atom'` line underneath.

File: Casks/formulatepro.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  name 'formulatepro'
```
Comment (vitorgalvao) [inline Casks/formulatepro.rb]: Why not `FormulatePro`? Capitalisation matters, in this field.

**Comment by vitorgalvao** [comment]
This needs some fixes, since it is a fork. @fanquake This is the kind of stuff we need to care about: the `homepage` and `url` are from different users, and that matters a lot. In this case it is completely safe, but someday it might not be, and we have to always check those.

**Comment by shimeike** [comment]
> Has to be changed to shimeike-formulatepro, to make it clear it is a fork.
> Why not FormulatePro? Capitalisation matters, in this field.

Does the cask filename therefore have to be changed to match as well (i.e. shimeike-FormulatePro.rb)? Assume that my branch can still be named 'formulatepro'?

**Comment by vitorgalvao** [comment]
> Does the cask filename therefore have to be changed to match as well

Yes.

> i.e. shimeike-FormulatePro.rb

No. `shimeike-formulatepro.rb`.

> Assume that my branch can still be named 'formulatepro'?

Yes.

**Comment by shimeike** [comment]
ok, done.

Thanks for your help, guys.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**